setup/operators/operators.go

```diff
@@ -91,28 +91,47 @@ func EnsureOperatorsInstalled(cl client.Client, s *runtime.Scheme, templatePaths
 			return err
 		}
 
+		startTime := time.Now()
+
 		// wait for operator installation to succeed
 		var csverr error
 		var currentCSV string
+		var lastCSVs []string
 		err = wait.ForSubscriptionWithCriteria(cl, subscriptionResource.GetName(), subscriptionResource.GetNamespace(), func(subscription *v1alpha1.Subscription) bool {
 			currentCSV = subscription.Status.CurrentCSV
 			if currentCSV == "" {
 				return false
 			}
-			// waiting for csv should fail quickly so that the currentCSV can be reloaded in case it was changed
+
+			if len(lastCSVs) == 0 || currentCSV != lastCSVs[len(lastCSVs)-1] { // subscription's current CSV has changed
+				lastCSVs = append(lastCSVs, currentCSV)
+				fmt.Printf("CurrentCSV of subscription: '%s'\n", currentCSV)
```

Comment:
We could use some logger for nicer output.

Reply:
Yeah, I should probably do that 🙂 I think I'll do it in a separate PR